ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -233,7 +235,11 @@ private[sandbox] final class InMemoryLedger(
               LedgerEntry.Rejection(recordTime, commandId, `appId`, submissionId, actAs, reason)
             ),
           ) if actAs.exists(parties) =>
-        val status = reason.toParticipantStateRejectionReason.status
+        val status = reason
+          .toParticipantStateRejectionReason(errorFactories)(
+            new DamlContextualizedErrorLogger(logger, loggingContext, None)
```

Comment:
Same question as earlier. Can/should we pass the `submissionId` to the logger?

Reply:
Done 87c9b918072712fed1b6224060a2b39f34cf6feb